src/js/components/app.js

```diff
@@ -18,6 +19,19 @@ import { getOnboardingComponentFor } from '../utils/onboardingmanager';
 const timeout = 900000; // 15 minutes idle time
 
 class AppRoot extends React.PureComponent {
+  componentDidMount() {
+    this.props.history.listen(location => {
+      let page = location.pathname || '';
+      ReactGA.set({ page });
```

Comment:
Why do you need this set?

Reply:
that should act as a https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#dimension - I actually forgot to assign this with the stripped down path... should now make more sense I hope

Reply:
@mzedel yup, but why do you need to define the page as custom dimension? It is already tracked as URL by pageView.

Reply:
hmm that was supposed to track the context area/ tab the user is working with - I thought the pageview tracking was not automatically splitting this up but if it does I'll happily remove it...

Reply:
@mzedel I confirm you when you call `ReactGA.pageview(page);` the page variable, which contains the URL, is available in Google Analytics. No need to track it as a custom dimension.

Reply:
thanks for the clarification - should now be removed 👌!